packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -121,13 +126,8 @@ const pageEvents = [];
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout(PUPPETEER_TIMEOUT || 100000);
 
-// Retry flaky tests at most 2 times in CI (off by 1).
-if ('true' === process.env.CI) {
```

Comment:
This can't be set at the global level

https://jestjs.io/docs/jest-object#jestretrytimesnumretries-options

> This must live at the top-level of a test file or in a describe block.